src/main/java/frc/robot/subsystems/swerve/states/SwerveState.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot.subsystems.swerve.states;
```

Comment:
Maybe support logging of this class?

Reply:
I was doing it in swerve but I’ll move it here